groclient/lib.py

```diff
@@ -675,7 +704,8 @@ def get_descendant(access_token, api_host, entity_type, entity_id, distance=None
         else:
             params['distance'] = -1
 
-    params['includeHistorical'] = include_historical
+    if entity_type == 'regions':
```

Comment:
This is the fix - metrics and items should not set the `includeHistorical` flag.